documentation/src/main/asciidoc/user_guide/transactions.adoc

```diff
@@ -244,11 +244,11 @@ Pessimistic transactions are more costly by their nature: each write operation p
 
 Write skews occur when two transactions independently and simultaneously read and write to the same key. The result of a write skew is that both transactions successfully commit updates to the same key but with different values.
 
-The `write-skew` attribute automatically detects and rolls back one of the transactions. However, write skew checks apply only to `REPEATABLE_READ` isolation levels in optimistic transactions.
+Write skew checking is disabled except for {brandname} Library Mode, which defaults to the `REPEATABLE_READ` isolation level with optimistic locking transactions.
 
 [NOTE]
 ====
-The `write-skew` attribute is deprecated.
+`write-skew` is a {brandname} configuration attribute that detects and rolls back transactions. However, the `write-skew` attribute is deprecated. See the configuration schema for more information.
```

Comment:
What about: `The write skew attribute is deprecated. Infinispan automatically sets it to ensure data consistency`?

Reply:
updated as discussed